android/app/build.gradle

```diff
@@ -59,4 +59,5 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation platform('com.google.firebase:firebase-bom:26.6.0')
     implementation 'com.google.firebase:firebase-analytics'
+    mplementation 'androidx.browser:browser:1.2.0'
```

Comment:
It's not correctly added, and it's not needed in the current implementation as there is no OTP verification in the repo currently :)